### DISCUSSION TITLE
Wakapi @ Twitter

### DISCUSSION THREAD

**Discussion by muety** [muety/wakapi#363] [General]
Many open source tools nowadays have their own Twitter page, including https://twitter.com/PirschAnalytics, https://twitter.com/caddyserver, https://twitter.com/prisma, https://twitter.com/litestreamio and many more. I thought about this for Wakapi, too, but would like to hear you guys' opinion.

If Wakapi was about to have its own Twitter page, **would you follow it** and **what would you want to see there**?

Content ideas:
1. Release notifications / changelogs
1. Tech deep dives
    * E.g. when I deeply went into debugging and gained some interesting insights from that (e.g. https://github.com/muety/wakapi/issues/350#issuecomment-1097276752, https://github.com/muety/wakapi/issues/340#issuecomment-1072376146, https://github.com/muety/wakapi/issues/294#issuecomment-1004386206, ...)
1. Articles and tips about (web-) dev in general
1. Shoutouts to other cool projects
1. (Anonymized) statistics / summaries of coding time on Wakapi.dev
1. Leaderboard updates (once #182 is implemented)
1. ... you name it ...

**Comment by mawoka-myblock** [comment]
Great idea, but please consider a Mastodon-account instead of a Twitter-Account...
